src/test/resources/task2/flowers.xsd

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```

Comment:
I assume it is the same as main/resources/task2/flowers.xsd.
So it shouldn't be duplicated here. This file can be deleted